aim/sdk/run.py

```diff
@@ -636,6 +608,77 @@ def _get_sequence(
         sequence = seq_cls(sequence_name, context, self)
         return sequence if bool(sequence) else None
 
+    def _prepare_sequence_info(self, read_only):
+        if read_only:
+            return
+
+        try:
+            for ctx_id in self.meta_tree['contexts'].keys():
```

Comment:
please check if its possible to use `self.meta_run_tree.subtree('traces').items() to iterate over contexts/traces.

Reply:
good point. yes, we can use `self.meta_run_tree.get('traces', {}).items()`